src/data/__tests__/cookingTimes.test.js

```diff
@@ -1,8 +1,8 @@
 /* global test, describe, it, expect */
 'use strict'
-import cooking-times from '~/GroceriStar/cd/cooking-times.js'
+import cooking-times from '../../../cooking-times'
```

Comment:
please change cooking-times on cooking_times because filenames and wariable shouldn't contain -(dash).
also change filenames, object and export